lib/tailwind.ex

```diff
@@ -105,18 +107,36 @@ defmodule Tailwind do
   # Latest known version at the time of publishing.
   def latest_version, do: @latest_version
 
+  @doc false
+  def profiles do
+    config_keys = [:version_check, :version, :target, :path]
+    :tailwind |> Application.get_all_env() |> Keyword.drop(config_keys)
```

Comment:
And we should probably filter it to only return entries which have keyword lists as values.

```suggestion
    :tailwind |> Application.get_all_env() |> Keyword.drop(config_keys)
```